### ISSUE TITLE
JAWS is not announcing "Group Color" label name when color radio buttons receiving focus: A11y_MySQL extension for Azure Data Studio_Create new server group_ScreenReader.

### ISSUE THREAD

**Issue by Swati1700** [microsoft/azuredatastudio-mysql#234]
"[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”
**Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification.** 

### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #BM_MySQL extension for Azure Data Studio_Win32_Mar2024; #MySQL extension for Azure Data Studio; #Win32; #Win11; #JAWS; #ScreenReader; #WCAG1.3.1; #InfoandRelationships; #Element:Label; #FTP; #External; External:ADS; #FirstParty:Bug25777; 

### Environment and OS details:
Azure Data Studio Version: 1.48.0 (user setup)
Windows 11 pro Version 23H2
OS build 22631.3296
Screen Reader: JAWS 2024, NVDA 2023.3.2

### Repro Steps:

1. Launch Azure Data Studio.
3. Start JAWS.
4. Tab to the Connections icon and press enter to Select (if not already selected) from left bar.
5. Tab to the icon that has a tool tip of 'New Server Group' icon and press enter.
6. Enter the name for the Server group name and a description. 
7. Tab to "Group color" section radio buttons.
8. Observe JAWS is not announcing "Group Color" label name when color radio buttons receiving focus.

### Actual Result:
JAWS is not announcing "Group Color" label name when color radio buttons receiving focus.

**Note: This issue does not repro with NVDA.**

### Expected Result:
JAWS should be announcing the "Group Color" label name when color radio buttons receiving focus.

### User Impact:
Screen reader users will not be able to understand the purpose of the color radio buttons and will get confused.

### Attachment:
[JAWS.zip](https://github.com/microsoft/azuredatastudio-mysql/files/14718628/JAWS.zip)
[NVDA_Norepro.zip](https://github.com/microsoft/azuredatastudio-mysql/files/14718648/NVDA_Norepro.zip)
Code snippet:
![Screenshot (1615)](https://github.com/microsoft/azuredatastudio-mysql/assets/86710888/a5d2854d-3489-48dd-8afe-9b6769bd03fc)

**Comment by nitishsharma25** [comment]
hi @Swati1700, this interface is common to all extensions in ADS, I checked this behavior is common for SQL as well as MySQL, maybe the correct place to raise this issue is in azuredatastudio repo.

**Comment by Swati1700** [comment]
Logged external bug in azuredatastudio repo: https://github.com/microsoft/azuredatastudio/issues/25566

**Comment by nitishsharma25** [comment]
Hi @Swati1700 , any update on this?

**Comment by Swati1700** [comment]
Logged external to ADS bug: https://github.com/microsoft/azuredatastudio/issues/25777 
We will act on this bug once the external bug is fixed and closed. As per the C & AI bug process assigning the bug to the Product Owner. Thanks!

**Comment by Swati1700** [comment]
Verified the issue in the environment 
ADS Version: 1.50.0 (user setup)
Issue is still reproducing. Please find the repro attachment below:
https://github.com/user-attachments/assets/663078b9-238d-4284-8453-96a678d975fa

**Comment by kupatkar99** [comment]
GithubTags:#ResolvedByTT;

**Comment by rishky1001** [comment]
Closing this bug as external bug is closed.
